### PR TITLE
Align values with cluster-openstack

### PR DESCRIPTION
**Pull request by tfussell**
This PR:

- Aligns the structure of values.yaml to match [cluster-openstack](https://github.com/giantswarm/cluster-openstack/blob/main/helm/cluster-openstack/values.yaml) so the same values file can be used for both apps.
- Remove configurable app-operator version label made unnecessary by https://github.com/giantswarm/app-operator/pull/902

### Testing

- [x] Fresh install works.

### CODE REVIEW

File: helm/default-apps-openstack/templates/_helpers.tpl [hunk 1]
```diff
@@ -17,32 +17,32 @@ Create chart name and version as used by the chart label.
 Common labels
 */}}
 {{- define "labels.common" -}}
-app-operator.giantswarm.io/version: {{ .Values.appOperator.version }}
+app-operator.giantswarm.io/version: 5.5.1
```
Comment (tfussell) [inline helm/default-apps-openstack/templates/_helpers.tpl:20]: No longer needs to be configurable, just needs to be a non-empty value and not 1.0.0 (see https://github.com/giantswarm/app/blob/master/pkg/validation/app.go#L259-L264). Will remove once the validation is improved.

Comment (rossf7) [reply helm/default-apps-openstack/templates/_helpers.tpl:20]: Thanks @tfussell I created https://github.com/giantswarm/giantswarm/issues/20448 to improve the validation logic.

**Review by rossf7 — APPROVED** [review]
LGTM

**Review by kopiczko — APPROVED** [review]
Yes please :)

**Pull request merged**